source/docs/articles/sites/code/hot-fixes.md

```diff
@@ -2,8 +2,10 @@
 title: Hot Fixes
 description: Learn how to deploy a hotfix on Pantheon.
 category:
-  - developing
+  - developing
 ---
+For Experts only. You should not need to attempt this if you use [Multidev](/docs/articles/sites/multidev) and keep commits from reaching Dev that you do not intend on deploying.
+
 Sometimes it's necessary to push a quick fix without pushing everything that's been going on in dev. This is called a "hotfix", and this article will help you do it on Pantheon.

```

Comment:
@ari-gold I agree with your proposed change. Go ahead and replace the line with the comments of your comment.